[EDU-6046] Create Azion Lib docs in English (#1479)
aziontech/docs: src/content/docs/en/pages/devtools/azion-lib/lib-overview/overview.mdx

---
title: Azion Libraries
description: >-

>   Learn about the Azion Libraries, a suite of tools for interacting with Azion services, products and utilties.

Reviewer: ```suggestion
  Learn about Azion Libraries, a suite of tools for interacting with Azion services, products, and utilities.
```

>   Learn about the Azion Libraries, a suite of tools for interacting with Azion services, products and utilties.

Reviewer 2: ```suggestion
  Learn about the Azion Libraries, a suite of tools for interacting with Azion services, products, and utilities.
```

permalink: /documentation/products/azion-lib/overview/
meta_tags: >-
  Azion Lib, edge computing, edge applications, API integration, Azion products
namespace: documentation_azion_lib_overview
---

import LinkButton from 'azion-webkit/linkbutton'

> The **Azion Libraries** provide a suite of tools to interact with various Azion services, including Products (for example, **Purge** and **Storage**) and Utilities (like **Image Processor** and **Cookies**). Each library is configurable and supports debug mode and environment variable-based configuration.

Reviewer: ```suggestion
The **Azion Libraries** provide a suite of tools to interact with various Azion services, including products (for example, **Purge** and **Edge Storage**) and utilities (like **Image Processor** and **Cookies**). Each library is configurable and supports debug mode and environment variable-based configuration.
```

Reviewer: Check with PM and the Product Catalog

These libraries are designed to be versatile and can be used both within and outside of the [Azion Runtime](/en/documentation/products/azion-edge-runtime/) environment. When used outside of the Azion Runtime, the libraries will interact with Azion services via REST APIs. However, when used within the Azion Runtime, the libraries will leverage internal runtime capabilities for enhanced performance and efficiency.

---

## Installing Azion Libraries

> You can install the package using **NPM** or **Yarn**:

Reviewer: ```suggestion

You can install the package using **NPM** or **Yarn**:
```

```bash
npm install azion
```

```bash
yarn add azion
```

> 
> ## Using Azion Libraries

Reviewer: ```suggestion

---

## Using Azion Libraries
```

> For more information about how to use the **Azion Libraries**, check the [documentation on NPM](https://www.npmjs.com/package/azion).

Reviewer 2: ```suggestion
For more information about how to use **Azion Libraries**, check the [documentation on NPM](https://www.npmjs.com/package/azion).
```

> <LinkButton link="/en/documentation/products/azion-lib/client" label="Learn more about Azion Libraries `Client`" severity="secondary" target="_blank" />
> <LinkButton link="/en/documentation/products/azion-lib/config" label="Learn more about Azion Libraries `Config`" severity="secondary" target="_blank" />

Reviewer: ```suggestion
<LinkButton link="/en/documentation/products/azion-lib/client" label="Go to Azion Libraries `Client`" severity="secondary" target="_blank" />
<LinkButton link="/en/documentation/products/azion-lib/config" label="Go to Azion Libraries `Config`" severity="secondary" target="_blank" />
```
